### PR TITLE
Update to use android-complete version files

### PR DESCRIPTION
**Pull request by shoatman**
Please make sure every Pull Request has the following information:

* A reference to a related issue in your repository (mandatory)
* A description of the changes proposed in the pull request
* @mentions of the person required for reviewing proposed changes (optional)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,53 +1,50 @@
...
-  - emulator -avd test -no-audio -no-window &
-  - android-wait-for-emulator
-  - adb shell input keyevent 82 &
+  - $ANDROID_HOME/platform-tools/adb wait-for-device
+  - $ANDROID_HOME/platform-tools/adb shell input keyevent 82 & 
 
 script:
   - cd $PWD
-  - travis_wait ./gradlew clean adal:assembleLocal adal:connectedLocalDebugAndroidTest -PdisablePreDex
+  - travis_wait ./gradlew clean adal:assembleLocal adal:connectedLocalDebugAndroidTest -PdisablePreDex
```
Comment (AdamBJohnsonx) [inline .travis.yml:50]:
```suggestion
  - travis_wait ./gradlew clean adal:assembleLocal adal:connectedLocalDebugAndroidTest -PdisablePreDex
  
```

File: .travis.yml [hunk 2]
```diff
@@ -1,53 +1,50 @@
...
+  - sudo -E sudo -u $USER -E bash -c "$ANDROID_HOME/emulator/emulator -avd test -no-audio -no-window &"
 
 before_script:
-  - android list targets
-  - echo no | android create avd --force -n test -t android-$EMULATOR_API_VERSION --abi armeabi-v7a
-  - emulator -avd test -no-audio -no-window &
-  - android-wait-for-emulator
-  - adb shell input keyevent 82 &
+  - $ANDROID_HOME/platform-tools/adb wait-for-device
+  - $ANDROID_HOME/platform-tools/adb shell input keyevent 82 &
```
Comment (shahzaibj) [inline .travis.yml:46]: Could these sections of the travis files be loaded from specific place? may be in common? as these seem to be mostly same across repos

File: gradle/versions.gradle [hunk 3]
```diff
@@ -23,19 +26,40 @@ ext {
...
+    projectVersionName = "1.5.9"
+
+    junitLibraryVersion = "4.12"
+    androidSupportLibraryVersion = "28.0.0"
+    gsonLibraryVersion = "2.8.0"
+    supportTestLibraryVersion = "0.4.1"
+    mockitoLibraryVersion = "1.2"
+    dexMakerLibraryVersion = "1.2"
+    //#### The ABOVE are TSL version variables - END
 }
```
Comment (AdamBJohnsonx) [inline gradle/versions.gradle:65]:
```suggestion
}

```

File: gradle.properties [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
 
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
+# This the TSL versionName...
+versionName=1.5.9
+
+# For OneAuth default abiSelection
+abiSelection=x86
```
Comment (AdamBJohnsonx) [inline gradle.properties:18]:
```suggestion
abiSelection=x86

```

File: .travis.yml (hunk 2 above)
Comment (shoatman) [reply .travis.yml:46]: I'm planning to change the android_complete update versions scripts I wrote to do this... travis expects a file in the specific repo at a specific location so i'm guessing i can't use a specific file.

File: gradle/versions.gradle [hunk 5]
```diff
@@ -3,15 +3,18 @@ ext {
...
     minSdkVersion = 16
     automationAppMinSDKVersion = 21
-    targetSdkVersion = 27
+    targetSdkVersion = 28
     compileSdkVersion = 28
     buildToolsVersion = "28.0.3"
 
     // Plugins
-    gradleVersion = '3.3.1'
+    gradleVersion = '4.0.1'
```
Comment (AdamBJohnsonx) [inline gradle/versions.gradle:11]: Is this the Gradle version or the Gradle plugin version?  The Gradle version should be 6.1.1, right?

**Comment by AdamBJohnsonx** [comment]
Did we want to run ./gradlew --gradle-version 6.1.1?

**Comment by shoatman** [comment]
> Did we want to run ./gradlew --gradle-version 6.1.1?

It's run by gradle itself prior to executing the command.

File: gradle/versions.gradle (hunk 5 above)
Comment (shoatman) [reply gradle/versions.gradle:11]: gradle version

**Comment by AdamBJohnsonx** [comment]
> > Did we want to run ./gradlew --gradle-version 6.1.1?
> 
> It's run by gradle itself prior to executing the command.

Do we want to prepopulate the gradle-wrapper.jar, I suppose is another way of asking it.

**Comment by shoatman** [comment]
> > > Did we want to run ./gradlew --gradle-version 6.1.1?
> > 
> > 
> > It's run by gradle itself prior to executing the command.
> 
> Do we want to prepopulate the gradle-wrapper.jar, I suppose is another way of asking it.

What would be the advantage of doing that?

**Review by AdamBJohnsonx — DISMISSED** [review]
I don't have any real problems, just odd questions.

**Comment by AdamBJohnsonx** [comment]
> > > > Did we want to run ./gradlew --gradle-version 6.1.1?
> > > 
> > > 
> > > It's run by gradle itself prior to executing the command.
> > 
> > 
> > Do we want to prepopulate the gradle-wrapper.jar, I suppose is another way of asking it.
> 
> What would be the advantage of doing that?

Build predictability, I would think.  We currently ship this jar around anyway - why not ship a newer one?  Can we remove it from our packaging?

**Review by shahzaibj — DISMISSED** [review]

**Pull request closed without merging**